### PR TITLE
Microverse admissions Trial  project review

### PR DESCRIPTION
**Pull request by MohammedElseddik**
## Microverse admissions Trial  project

in this project  I:

- Reviewed my Microverse admission trial project .
- followed the code review instruction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Microverse-code-review-
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (MohammedElseddik) [inline README.md:5]: Please  add more section to README.md file  and make more descriptive, Add section like project description, Author, Acknowledgements.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+    background-color: rgb(253, 253, 253);
+    border-radius: 5px;
+  }
+  
+  .navbar-li a:hover {
+    cursor: pointer;
+    color: crimson;
+  }
+  
+  /* about-section */
```
Comment (MohammedElseddik) [inline css/style.css:70]: Please do not leave unused code as comments in your codebase, It makes your code not clean.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+    height: 100vh;
+    background-color: black;
+    color: white;
+    font-size: 2rem;
+    font-weight: bold;
+    border-bottom: 10px solid crimson;
+  }
+  
+  .about-section i {
+    color: crimson;
```
Comment (MohammedElseddik) [inline css/style.css:87]: Please consider using CSS attribute instead of using this color multiple times.

**Review by MohammedElseddik — COMMENTED** [review]
## Required Changes ♻️

- [ ] Kindly check the comments.
- [ ] Kinley document your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) follow the rules in this link.
- [ ] Please add linter to your project and fix linter errors.

## Optional suggestions

- Sort CSS properties. This makes it easier to read your code.
- Use CSS variables.

**Review by BenMukebo — APPROVED** [review]
## Hi @MohammedElseddik, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**